src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+    - value: a positive integer
+    - interval: one minute, one hour, one day, one week, or 30 days
+- A new event is seen
+- A new event is seen with tags matching {key} {match} {value}
+    - key: any tag [LINK: tags]
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any key’s value
+- A new event is seen with {attribute} {match} {value}
+    - attribute: `message`, `platform`, `environment`, `type`, `exception.type`, `exception.value`, `user.id`, `user.email`, `user.username`, `user.ip_address`, `http.method`, `http.url`, `stacktrace.code`, `stacktrace.module`, or `stacktrace.filename`
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any attribute’s value
+- A new event is seen with level {match} {level}
+    - match: equal to, less than or equal to, greater than or equal to
+    - level: fatal, error, warning, info, debug, or sample
+
+### **Actions**
+
+The following actions are available:
+
+- Send a notification to all legacy integrations
+    - Legacy integrations [LINK: legacy integrations], also known as Plugins, are configured per project. Legacy integrations currently include email.
+- Send a notification to a single legacy integration (service)
+    - "service" is the same as legacy integration
+- Send a notification to a global integration, which includes
+    - [PagerDuty](https://docs.sentry.io/workflow/integrations/global-integrations/#pagerduty)
+    - [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack)
+    - Custom integrations built using the [Integration Platform](https://docs.sentry.io/workflow/integrations/integration-platform/)
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png)
+
+- Legacy Integrations (via “Service” as any individual integration) that can send Notifications:
+
+    [ ALERT BOX: **Community Integrations** 
+
+    These integrations are [maintained and supported](https://forum.sentry.io/) by the Sentry community. ]
+
+    - Campfire*
+    - IRC
+    - OpsGenie
+    - Pushover
+    - Twilio
+    - VictorOps
+    - [Webhooks](https://docs.sentry.io/webhook-plugin/)
+
+By default, **alert emails** are sent to [LINK: issue owners]. If issue owners isn't configured, or an owner isn't found, the email will do one of two things:
+
+- not send
+- send to all project members depending on the fall-through setting under **Project Settings > Issue Owners**.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png)
+
+**Rate Limit**
+
+This control allows you to limit the number of times actions are executed for each issue.
```

Comment:
Can we combine these into one paragraph? 

**Rate Limit** control allows you to limit the number of times actions execute for each issue. The limit is set to perform the action *at most* according to one of these intervals: 
- minutes: 5, 10, 30, 60
- hours: 3, 12, 24
- one week or 30 days

Or something similar? We seem to use a great deal of white space and it would be nice to group like intervals, I think.